charts/baremetal-operator/templates/ironic.yaml

```diff
@@ -81,6 +81,8 @@ spec:
               value: {{ .Values.ironic.ironic_conf.provisioning_interface }}
             - name: HTTP_PORT
               value: {{ .Values.ironic.ironic_conf.http_port | quote }}
+            - name: IRONIC_FAST_TRACK
+              value: "{{ .Values.ironic.ironic_conf.ironic_fast_track }}"
```

Comment:
use `.Values.ironic.ironic_conf.ironic_fast_track | quote` instead of double quotes like it's implemented in HTTP_PORT variable above